array-manipulation-2/src/take-right.js

```diff
@@ -0,0 +1,19 @@
+/* exported takeRight */
+/*
+  set output equal to empty array
+  set i equal to array to different of array length and count
+  while i is  less then array length minus count
+    append array at i to output
+  return output
+*/
+
+function takeRight(array,count){
+  var output = [];
+  if(count > array.length){
+    return [];
```

Comment:
In this case, if the count is more than the size of the array, the entire array should be given back since that would mean you are "taking" all the data inside of it.